src/constructors.jl

```diff
@@ -249,7 +249,7 @@ julia> val = GAP.evalstr("[ true, false, true ]")
 GAP: [ true, false, true ]
 
 julia> BitArray{1}(val)
-3-element BitArray{1}:
+3-element BitVector:
```

Comment:
Strange: This is the only place where `BitArray{1}` got changed to `BitVector`. Why did the replacements find this one but not the others?

Reply:
Just to record in public what I mentioned in private chat: this second commit was made by running `doctest(GAP, fix=true)`.